index.bs

```diff
@@ -54,17 +54,11 @@ The following keys must be considered valid:
 * <a href="#key-content_scripts">`content_scripts`</a>: optional
 * <a href="#key-content_security_policy">`content_security_policy`</a>: optional
 * <a href="#key-description">`description`</a>: optional
-* <a href="#key-devtools_page">`devtools_page`</a>: optional
-* <a href="#key-externally_connectable">`externally_connectable`</a>: optional
```

Comment:
Safari has an implementation of `externally_connectable` and `devtools_page`. So those items should remain.

Reply:
Resolved with latest commit.